ReadMe.md

```diff
@@ -31,11 +31,13 @@ npm install --global markdownlint-cli2
 ```
 
 
-## Oxypng
+## Oxipng
 
-We use [oxpng](https://github.com/shssoichiro/oxipng) to optimize image file size.
-You can install it with ``cargo install oxipng``
-and then run it with ``oxipng --strip safe --alpha -r src``.
+We use [oxipng](https://github.com/shssoichiro/oxipng) to optimize image file size.
+You can install it with `cargo install oxipng` and then run it as follows:
+
+```bash
+oxipng --strip safe --alpha -r src
```

Comment:
As mentioned, it wasn't rendered properly -- the bash codeblock still needs a closing triple backtick 🙂

Reply:
I did add closing ticks with another commit, or do you mean something else? ![image](https://github.com/user-attachments/assets/68145c78-3885-4fad-9c82-cea7ca0f5639)

Reply:
You're right! For some reason this didn't display in my diff. Sorry!